panels/nerf_3dview_panels/dataset_panel.py

```diff
@@ -33,15 +37,20 @@ def unregister(cls):
         """Unregister properties and operators corresponding to this panel."""
         bpy.utils.unregister_class(ImportNeRFDatasetOperator)
         bpy.utils.unregister_class(ExportNeRFDatasetOperator)
+        bpy.utils.unregister_class(RemoveNeRFDatasetOperator)
         cls.remove_observers()
 
 
 
-    def draw(self, context):
-        """Draw the panel with corresponding properties and operators."""
-
-        layout = self.layout
+    def dataset_section(self, context, layout, ui_props):
```

Comment:
why was this changed?

Reply:
To try and see how I can alter the training panel. Chances are that defining can be reverted